commands.go

```diff
@@ -710,6 +721,35 @@ func (c cmdable) GetSet(ctx context.Context, key string, value interface{}) *Str
 	return cmd
 }
 
+// redis-server version >= 6.2.0.
+//
+// A value of zero means that the expiration time will not be changed.
+// Persist(-2) Remove the time to live associated with the key.
+func (c cmdable) GetEX(ctx context.Context, key string, expiration time.Duration) *StringCmd {
```

Comment:
I think `GETEX` means `GET` + `EX[PIRE]` which should be `GetEx` in Go. What about calling this `GetExpire`? It is unfortunate to intrudoce a different name for Redis command, but it is confusing to have the names like `SetNX` and `GetEX` which mean completely different things...

I don't think we need `redis.Persist` at, because in `Set` we use `0` to persist the key, .e.g it is `Set(key, value, 0)`, not `Set(key, value, redis.Persist)`. So passing `0` should mean `PERSIST`.